packages/integration-karma/test/misc/performance-timing/index.spec.js

```diff
@@ -177,19 +177,29 @@ if (isUserTimingSupported) {
                                 label: /<x-child> - constructor/,
                             },
                             {
-                                label: /<x-child> - render/,
-                            },
-                            {
-                                label: /<x-child> - patch/,
+                                label: /lwc-hydrate/,
```

Comment:
Adding marks on all the components in production is really worrying. On top of flooding the server with a lot of marks, it will degrade performance. Let's discuss how we want to approach this.

Reply:
After doing some local testing, adding performance marks for all the component does introduce a 34% performance regression on some of the most CPU intensive tests. It's important to note that it is measured with the profiler attached. Even if attaching the profiler increase the cost each individual marks, adding all those marks has a measurable performance impact.

FYI @caridy @ravijayaramappa